packages/web-react/src/components/Modal/demo/ModalScrollingLongContent.tsx

```diff
@@ -151,26 +195,115 @@ const ModalScrollingLongContent = () => {
 
       <Modal id="example-custom-height" isOpen={isModalCustomHeightOpen} onClose={handleModalCustomHeightClose}>
         <ModalDialog
+          height={generateHeightObject()}
           isExpandedOnMobile={false}
           isScrollable
-          maxHeightFromTabletUp="700px"
-          preferredHeightOnMobile="400px"
-          preferredHeightFromTabletUp="500px"
+          maxHeight={generateHeightObject(true)}
+          ref={modalCustomHeightRef}
         >
           <ModalHeader>Modal with Custom Height</ModalHeader>
           <ModalBody>
-            <p className="d-tablet-none">
-              This modal has a custom height of <code>400px</code>.
-              <br />
-              <br />
-              The max height cannot be customized on mobile though.
-            </p>
-            <p className="d-none d-tablet-block">
-              This modal has a custom height of <code>500px</code>.
-              <br />
-              <br />
-              The max height of this modal is <code>700px</code>.
-            </p>
+            <form>
+              {Object.entries(breakpointControls).map(([breakpoint, { marginBottom, className }]) => (
+                <Stack
+                  elementType="fieldset"
+                  hasSpacing
+                  key={breakpoint}
+                  marginBottom={marginBottom as Partial<Record<string, `space-${number}`>>}
+                  UNSAFE_className={className}
+                  UNSAFE_style={{ border: 0 }}
+                >
+                  <legend hidden>{breakpoint.charAt(0).toUpperCase() + breakpoint.slice(1)}</legend>
+                  <Checkbox
+                    id={`custom-height-${breakpoint}-enabled`}
+                    isChecked={isCustomHeightEnabled[breakpoint]}
+                    label={breakpoint.charAt(0).toUpperCase() + breakpoint.slice(1)}
+                    onChange={(event) =>
+                      setIsCustomHeightEnabled((prevState) => ({
+                        ...prevState,
+                        [breakpoint]: (event.target as HTMLInputElement).checked,
+                      }))
+                    }
+                  />
+                  <Grid style={{ columnGap: 'var(--spirit-space-600)' }}>
```

Comment:
This is the why the layout jumps 🙂 

```suggestion
                  <Grid style={{ '--column-gap': 'var(--spirit-space-600)' }}>
```

Reply:
Nope, it has to be `UNSAFE_style`